scss/components/side-nav.scss

```diff
@@ -0,0 +1,113 @@
+@import "./../settings";
+@import "./../mixins";
+@import "./../functions";
+
+/*!
+.fd-side-nav
+    .fd-side-nav__group
+        .fd-side-nav__title
+        .fd-side-nav__list
+            .fd-side-nav__item
+                .fd-side-nav__link (.has-child, .is-active, .is-expanded)
+                .fd-side-nav__sublist
+                    .fd-side-nav__subitem
+                    .fd-side-nav__sublink (.is-active)
+*/
+
+$block: #{$fd-namespace}-side-nav;
+
+.#{$block} {
+
+    //LOCAL VARS
+    $fd-side-nav-background-color: fd-color("background", 2);
+    $fd-side-nav-selected-background-color: lighten( fd-color("action", 1), 52%);
+    $fd-side-nav-link-color: fd-color("text", 1);
+    $fd-side-nav-max-width: 250px;
+    $fd-side-nav-link-padding: fd-space(4);
+    $fd-side-nav-link-padding: fd-space("s");
+    $fd-side-nav-link-disabled-color: fd-color("text", 2);
+    $fd-side-nav-link-padding: fd-space("s");
+
+    //BLOCK BASE *******************************************
+    background-color: $fd-side-nav-background-color;
+    min-width: $fd-side-nav-max-width;
+    margin: 0;
+    padding: 0;
+
+    //ELEMENTS *******************************************
+    &__group {
+        list-style: none;
+        margin-bottom: 0;
+    }
+
+    &__title {
+        display: block;
+        margin-bottom: 0;
+        padding: fd-space(4);
+        padding-bottom: 0;
+        @include fd-type("-3", "body", "reg", uppercase);
+        color: $fd-side-nav-link-disabled-color;
+    }
+
+    &__list,
+    &__sublist {
```

Comment:
There is nothing here to hide this. In the examples, the sublist is visible but has `aria-hidden="true"`

```
<ul class="fd-side-nav__sublist" aria-hidden="true">
  <li class="fd-side-nav__subitem"><a class="fd-side-nav__sublink " href="#">
          Link
      </a>
  </li>
  <li class="fd-side-nav__subitem"><a class="fd-side-nav__sublink " href="#">
          Link
      </a>
  </li>
  <li class="fd-side-nav__subitem"><a class="fd-side-nav__sublink " href="#">
          Link
      </a>
  </li>
</ul>
```